bloom-housing/bloom
# feat: seeds button uptake

## Issue Overview

This PR addresses #3595 


## Description

This PR swaps out all UIC buttons with Seeds buttons in the Core applications. Note that this doesn't address any buttons which may be embedded within other UIC components.

## How Can This Be Tested/Reviewed?

Visit pages in the Public and Partners sites. Ensure that the buttons work and aren't broken. I noticed some translations have odd capitalization, which means you can now tell (because the old buttons were ALL CAPPS), but I would recommend addressing those in followup PRs.

**NOTE**: there are two Cypress tests for Partners which I cannot figure how to resolve. The error messages are confusing and don't make sense. Also note: the jest-shared-helpers test is failing, but that's due to an apparent TypeScript issue coming from Seeds, so if we want to resolve that we'll need to address in a separate issue/PR in that repo.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
